tests/integration/reqs/test_account_tx.py

```diff
@@ -13,3 +13,43 @@ async def test_basic_functionality(self, client):
             )
         )
         self.assertTrue(response.is_successful())
+
+    @test_async_and_sync(globals())
+    async def test_validate_response_type_v2(self, client):
+        response = await client.request(
+            AccountTx(account=WALLET.address, api_version=2)
+        )
+
+        # use the below proxies to ensure that the correct API response is returned
+        # API v2 returns tx_json field
+        self.assertTrue("tx_json" in response.result["transactions"][0])
+
+        # API v2 does not return a tx field
+        self.assertTrue("tx" not in response.result["transactions"][0])
+        self.assertTrue(response.is_successful())
+
+    @test_async_and_sync(globals())
+    async def test_validate_response_type_v1(self, client):
+        response = await client.request(
+            AccountTx(account=WALLET.address, api_version=1)
+        )
+
+        # use the below proxies to ensure that the correct API response is returned
+        # API v1 does not contain a tx_json field
+        self.assertTrue("tx_json" not in response.result["transactions"][0])
+
+        # API v1 returns a tx field
+        self.assertTrue("tx" in response.result["transactions"][0])
+        self.assertTrue(response.is_successful())
+
+    @test_async_and_sync(globals())
+    async def test_validate_response_type_no_explicit_api_version(self, client):
```

Comment:
Can the test names be a bit less verbose? e.g. 
```suggestion
    async def test_no_explicit_api_version(self, client):
```

Reply:
fixed in 394d1fd